packages/markdown/src/MarkdownEditor.tsx

```diff
@@ -29,14 +29,7 @@ export interface MarkdownEditorProps {
   isInitiallyDisabled: boolean;
 
   sdk: FieldExtensionSDK;
-  /**
```

Comment:
So we don't want to make it possible to disable the markdown editor being able to upload assets from an extension?

Reply:
Yeah, not at the moment. It was exposed only for Web App being able to pass the value.
We might get back to it and expose all buttons as instance parameters.